run_gigaspeech.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Looks like this file is specific to Kaldi, I'd suggesting we have a folder for toolkits/kaldi, and put Kaldi related scripts there, what do you guys think? @dophist @jimbozhang

Reply:
+1